doc/interfaces/dg.rst

```diff
@@ -0,0 +1,23 @@
+DG Interface
+================
+
+Interface to `DG Technologies <http://dgtech.com>`__ Beacon
+
```

Comment:
Are there any extra drivers required for this? What platforms (operating systems) does it run on?